modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContext.java

```diff
@@ -420,6 +421,18 @@ public boolean hasViewPermission() throws PortalException {
 		return _hasViewPermission;
 	}
 
+	public boolean isAutoRememberMe() {
```

Comment:
@joseabelenda, why did you use the name `auto` here? Can you please clarify?

Reply:
@natocesarrego, because I saw in file /portal-web/docroot/html/common/themes/session_timeout.jsp and decided to follow the same idea.